zdm/iteration.py

```diff
@@ -1020,16 +863,15 @@ def cube_likelihoods(grids,surveys, #psetmins,psetmaxes,npoints,
             if const_only:
                 C,llC,lltot=minimise_const_only(
                     vparams,grids,surveys)
-                #pset[7]=C
                 vparams['lC']=C
 
             
             # for the minimised parameters, set the values
             for j,n in enumerate(current):
-                #string +=' {:8.2f}'.format(pset[j])
                 string +=' {:8.2f}'.format(vparams[PARAMS[j]])
             
             
+            # TODO -- Should we do this?
```

Comment:
Hmm, need to think about this, I suspect not given my previous comment. Might need to have a parameter returned from a previous routine.

Reply:
I think it is ok and am leaving it for now.